test/compress/keep_names.js

```diff
@@ -88,6 +88,33 @@ keep_some_fnames: {
   }
 }
 
+keep_some_fnames_reduce: {
+  options = {
+    reduce_vars: true,
+    unused: true,
+    keep_fnames: /Element$/,
+  };
+  mangle = {
+    keep_fnames: /Element$/,
+  };
+  input: {
+    function foo() {
+      var array = []
+      function bar() {}
+      array.map(bar)
+      function barElement() {}
+      array.map(barElement)
+    }
+  }
+  expect: {
+    function foo() {
+      var n=[]
```

Comment:
Formatting - use space on either side of assign.

Preferably for test `input` and `expect` code terminate statements with semicolons unless it is exercising some bug in the parser.